File: python/dp-e01.py

def print_details(name, age, username):
    statement = "Your name is {}, you are {} years old and your username is {}"
    print(statement.format(name, age, username))

if __name__ == '__main__':
    name = input("Name >> ")
    age = int(input("Age >> "))
    username = input("Username >> ")

    print_details(name, age, username)


class MyClass(object):
    def __int__(self, name, owner, uid, breed, animal_type, likes_cats):
        self.name = name
        self.owner = owner
        self.uid = uid
        self.breed = breed
        self.animal_type = animal_type
        self.likes_cats = likes_cats

File: javascript/dp-e01.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Name >> ", (name) => {
  rl.question("Age >> ", (age) => {
    rl.question("Username >> ", (username) => {
      console.log(`Your name is ${name}, you are ${age} years old and your username is ${username}`);
    });
  });
});
